packages/patternfly-react/src/components/Filter/Filter.js

```diff
@@ -3,10 +3,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { getContext } from 'recompose';
 import { toolbarContextTypes } from '../Toolbar/ToolbarConstants';
+import FilterTypeSelector from './FilterTypeSelector';
+import FilterValueSelector from './FilterValueSelector';
+import FilterCategorySelector from './FilterCategorySelector';
+import FilterCategoryValueSelector from './FilterCategoryValueSelector';
+import FilterActiveLabel from './FilterActiveLabel';
+import FilterList from './FilterList';
+import FilterItem from './FilterItem';
 
 // Disabled eslint due to `isDescendantOfToolbar` being a context property we don't want passed by consumers
 // eslint-disable-next-line react/prop-types
-const Filter = ({ children, className, isDescendantOfToolbar, ...props }) => {
+const FilterContext = ({ children, className, isDescendantOfToolbar, ...props }) => {
```

Comment:
Nit. Naming is a little weird. What about `ContextualFilter`

Reply:
Agreed. Thanks for the suggestion, I couldn't come up with a name I liked.